.github/workflows/build-native.yml

```diff
@@ -2,28 +2,37 @@ name: Build Native Images
 on:
   release:
     types: [published]
+  pull_request:
 jobs:
   Build-Native-Images:
     strategy:
       matrix:
-        os: ["ubuntu-18.04", "macos-11"]
+        os: ["ubuntu-18.04", "macos-11", "odroid"]
     runs-on: ${{ matrix.os }}
     steps:
     - name: "Checkout FPP Repository"
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
+    - name: Set architecture type (aarch64)
+      if: ${{ matrix.os == 'odroid' }}
+      run: echo "architecture=aarch64" >> "$GITHUB_ENV"
+    - name: Set architecture type (amd64)
+      if: ${{ matrix.os != 'odroid' }}
+      run: echo "architecture=amd64" >> "$GITHUB_ENV"
     - name: Image Build Step
       working-directory: ./compiler
       run: |
-        . ${GITHUB_WORKSPACE}/.github/workflows/env-setup
+        . ${GITHUB_WORKSPACE}/.github/workflows/env-setup ${{ env.architecture }}
         ./release
-        mv native-fpp-*.tar.gz "${GITHUB_WORKSPACE}"
+        mv native-fpp-*.tar.gz "${GITHUB_WORKSPACE}" 
     - name: Publish Release Binaries
+      if: ${{ github.event_name == 'release' }}
```

Comment:
Since this is guarded, should we run the build on ever PR anyway? That way we'd know the native compile does not break.

Reply:
I think from previous discussion we talked about only running the section for publishing the release if the build was triggered via the "release" trigger. Are you referring to running this regardless of trigger type (so for PRs and releases)?

Reply:
> Since this is guarded, should we run the build on ever PR anyway? That way we'd know the native compile does not break.

I'd rather not run the native build on every PR during development. However, it would be nice to be able to test the native build near release time without creating a test release. Is there a middle ground here?

Reply:
Maybe we can include the release "pre-release" or "unpublished" triggers so that the build will run once a release is created (but not yet in the published state)?

Reply:
The native builds are a required feature of this code, and thus it seems that building natively should be a required test.  Is there a downside of this that I am not seeing?